### DISCUSSION TITLE
Filter events

### DISCUSSION THREAD

**Discussion by ModProg** [bitfireAT/icsx5#235] [Ideas]
I have a calendar from my local garbage collection, but I'm not interested in all collections.

I'd like to be able to have e.g. a regex/glob match that can filter events I want in my calendar and ignore the rest.

I'd look into implementing this if you'd consider including this feature.

**Comment by ArnyminerZ** [comment]
Hi, I think this should be done server-side, and not client-side. It's quite a complicated operation, and not so easy to implement, and I don't think it would be useful for general use. We also want to keep ICSx5 as simple as possible. However, if more people is interested in it, we do not discard anything. Hope you understand 😄

**Reply by ModProg** [reply]
Well, I am not sure, I can convince my local garbage collection company to build this feature or even let me add it, that's why I was thinking I could add it to icsx5 instead.

Maybe I'll just fork and add this feature, I might really be the odd one out needing this :D

**Reply by ArnyminerZ** [reply]
> Maybe I'll just fork and add this feature

That's also a really good option

**Reply by bb-froggy** [reply]
Any progress with the fork? 

My son's school publishes appointments (e.g. tests) for the whole school in one feed, but we are interested only in those of his class.

**Reply by ModProg** [reply]
I might have gotten distracted by other things, hoping to work on this soon™
